packages/react-components/react-datepicker-compat/src/components/CalendarDayGrid/CalendarDayGrid.tsx

```diff
@@ -163,7 +163,7 @@ export const CalendarDayGrid: React.FunctionComponent<CalendarDayGridProps> = pr
           weekIndex={-1}
           rowClassName={classNames.firstTransitionWeek}
           aria-role="presentation"
-          aria-hidden={true}
+          ariaHidden={true}
```

Comment:
Is there a specific reason for this change?

Reply:
The prop comes from CalendarDayGrid which has to be passed down all the way to CalendarDayGridCell, so since only CalendarDayGrid knows about 'aria-label' it has to be ariaLabel. Alternatively, we could add explicitly add aria-label and then we could remove ariaLabel.